File: python/1.py
def mults(n, start, max):
	return [i for i in xrange(start, max) if not i%n]

def multsArr(arr, start, max):
	return sum(set(m for n in arr for m in mults(n, start, max)))

# sum of all multiples of 3 or 5 up to 1000
print multsArr([3, 5], 0, 1000)

File: elixir/1.ex
defmodule Problem1 do
  @min 1
  @max 999
  @mults [3, 5]

  def sum_mult() do
    @min..@max
    |> Enum.filter(&is_mult(&1, @mults))
    |> Enum.sum()
  end

  defp is_mult(num, mults) do
    mults
    |> List.wrap()
    |> Enum.any?(&rem(num, &1) == 0)
  end
end